README.zh.md

```diff
@@ -0,0 +1,363 @@
+- **Multi-task**: The instruction tuning data and benchmark in PIXIU cover a diverse set of financial tasks.
+- **Multi-modality**: PIXIU's instruction tuning data and benchmark consist of multi-modality financial data, including time series data from the stock movement prediction task. It covers various types of financial texts, including reports, news articles, tweets, and regulatory filings.
+- **Diversity**: Unlike previous benchmarks focusing mainly on financial NLP tasks, PIXIU's evaluation benchmark includes critical financial prediction tasks aligned with real-world scenarios, making it more challenging.
+
+---
+
+## FLARE_ZH: Financial Language Understanding and Prediction Evaluation Benchmark
+
+In this section, we provide a detailed performance analysis of FinMA compared to other leading models, including ChatGPT, GPT-4, lince-zero et al. For this analysis, we've chosen a range of tasks and metrics that span various aspects of financial Natural Language Processing and financial prediction.
+
+### Tasks
+
+| Data                  | Task                            | Raw     | Data Types                       | Modalities        | License           | Paper |
+| --------------------- | ------------------------------- | ------- | -------------------------------- | ----------------- | ----------------- | ----- |
+| AFQMC                 | semantic matching               | 42,511  | question data, chat              | text              | Apache-2.0        | [1]   |
+| corpus                | semantic matching               | 120,000 | question data, chat              | text              | Public            | [2]   |
+| stockA                | stock classification            | 40,963  | news, historical prices          | text, time series | Public            | [3]   |
+| Fineval               | multiple-choice                 | 4,661   | financial exam                   | text              | Apache-2.0        | [4]   |
+| NL                    | news classification             | 10,000  | news articles                    | text              | Public            | [5]   |
+| NL2                   | news classification             | 10,000  | news articles                    | text              | Public            | [5]   |
+| NSP                   | negative news judgment          | 6,000   | news, social media text          | text              | Public            | [5]   |
+| RE                    | relationship identification     | 12,670  | news, entity pair                | text              | Public            | [5]   |
+| FE                    | sentiment analysis              | 10,000  | financial social media text      | text              | Public            | [5]   |
+| stockB                | sentiment analysis              | 270,601 | financial social media text      | text              | Apache-2.0        | [6]   |
+
+
+1. https://www.modelscope.cn/datasets/modelscope/afqmc_small/summary
```

Comment:
I find the paper here
https://autonlp.ai/datasets/ant-financial-question-matching-corpus-(afqmc)-(clue-benchmark)